include/ola/thread/Thread.h

```diff
@@ -33,18 +33,6 @@
 
 #include <string>
 
-#if defined(_WIN32) && defined(__GNUC__)
-inline bool operator==(const ptw32_handle_t &left,
-                       const ptw32_handle_t &right) {
-  return (left.p == right.p) && (left.x == right.x);
-}
-
-inline std::ostream& operator<<(std::ostream &stream,
-                                 const ptw32_handle_t &handle) {
-  stream << handle.p;
-  return stream;
-}
-#endif
```

Comment:
Yup, we should be using pthread_equal. The == operator isn't defined for pthread_t , see http://pubs.opengroup.org/onlinepubs/009695399/functions/pthread_equal.html